src/middlewares/headerMiddleware.js

```diff
@@ -0,0 +1,11 @@
+const headerConfig = {
+  assignmentNo: 3,
+  taskNo: 12,
+};
+
+const headerMiddleware = async (req, res, next) => {
```

Comment:
![image](https://github.com/ayush1-successive/backend-training/assets/83696224/785abef3-827a-4278-9a16-2705d23adf93)
This middleware is not working correctly